Dockerfile

```diff
@@ -191,8 +191,10 @@ COPY --from=rootfs-archive /rootfs.tar.gz /rootfs.tar.gz
 # The test target performs tests on the source code.
 
 FROM base AS test
-COPY --from=rootfs-build / /rootfs
-ENV PATH /rootfs/bin:$PATH
+#COPY --from=rootfs-build / /rootfs
```

Comment:
Let's stick with copying the rootfs. The only reason this works right now is because `containerd` is actually not linked against `libseccomp` (mistakenly), but that is fixed with the new `bldr` stuff I have been working on.